### ISSUE TITLE
Problem about finetune the DocRED dataset

### ISSUE THREAD

**Issue by WenxiongLiao** [thunlp/ERICA#11]
| epoch 19 | step 9700 |  ms/b 35103.27 | train loss 0.00095783 | NA acc: 0.99 | not NA acc: 0.95  | tot acc: 0.98 
| epoch 19 | step 9750 |  ms/b 1007.05 | train loss 0.00080489 | NA acc: 0.99 | not NA acc: 0.96  | tot acc: 0.98 
| epoch 19 | step 9800 |  ms/b 1034.59 | train loss 0.00080026 | NA acc: 0.99 | not NA acc: 0.96  | tot acc: 0.98 
| epoch 19 | step 9850 |  ms/b 1064.13 | train loss 0.00079512 | NA acc: 0.99 | not NA acc: 0.96  | tot acc: 0.98 
| epoch 19 | step 9900 |  ms/b 1235.63 | train loss 0.00084200 | NA acc: 0.99 | not NA acc: 0.96  | tot acc: 0.98 
| epoch 19 | step 9950 |  ms/b 1050.25 | train loss 0.00081005 | NA acc: 0.99 | not NA acc: 0.96  | tot acc: 0.98 
| epoch 19 | step 10000 |  ms/b 1057.81 | train loss 0.00076923 | NA acc: 0.99 | not NA acc: 0.96  | tot acc: 0.98 
| epoch 19 | step 10050 |  ms/b 1031.88 | train loss 0.00076228 | NA acc: 0.99 | not NA acc: 0.96  | tot acc: 0.98 
| epoch 19 | step 10100 |  ms/b 1010.93 | train loss 0.00079324 | NA acc: 0.99 | not NA acc: 0.96  | tot acc: 0.98 
| epoch 19 | step 10150 |  ms/b 1004.57 | train loss 0.00081850 | NA acc: 0.99 | not NA acc: 0.96  | tot acc: 0.98 
-----------------------------------------------------------------------------------------
dev set evaluation
ALL  : Theta 0.9079 | F1 0.5720 | AUC 0.5670
Ignore ma_f1 0.5501 | input_theta 0.9079 test_result F1 0.5494 | AUC 0.5376
**test set evaluation
ma_f1 0.0000 | input_theta 0.9079 test_result F1 0.0000 | AUC 0.0000
Ignore ma_f1 0.0000 | input_theta 0.9079 test_result F1 0.0000 | AUC 0.0000**

Finish training
Best epoch = 19

**Comment by thuqinyj16** [comment]
Could you print detailed information about the intermediate variables in the ### ``test'' function when evaluating on the test dataset?

**Comment by WenxiongLiao** [comment]
> Could you print detailed information about the intermediate variables in the ### ``test'' function when evaluating on the test dataset?

I output the variable ‘’_correct_‘’ in the test function, and I found that correct is not 0 in the validation set, while correct=0 in the test set.  What I'm curious about is why correct=0 in the test set?

**Comment by thuqinyj16** [comment]
It is generally hard to locate your problem here if you just provide a final output, could you please provide more detailed intermediate information so that we could help you figure our the problems here?
